css/css-view-transitions/parsing/view-transition-name-computed.html

```diff
@@ -22,6 +22,16 @@
 test_computed_value("view-transition-name", "inherit", "none");
 test_computed_value("view-transition-name", "revert", "none");
 test_computed_value("view-transition-name", "revert-layer", "none");
+
+// It seems the results are different in different browsers:
+// Gecko serializes this as "match-element",
+// WebKit doesn't serialize this, i.e. empty string.
```

Comment:
I prefer if specific results weren't mentioned in the tests since they'll be fixed, which makes it harder to import/export the test to WebKit, but no strong opinion

Reply:
Thanks. How about drop this PR and just land your version?

Reply:
https://github.com/web-platform-tests/wpt/pull/53391